src/claimtrie.h

```diff
@@ -28,6 +34,34 @@
 
 uint256 getValueHash(COutPoint outPoint, int nHeightOfLastTakeover);
 
+// lower-case and normalize any input string name
+// see: https://unicode.org/reports/tr15/#Norm_Forms
+inline std::string normalizeClaimName(const std::string& name)
+{
+    static std::locale utf8;
+    static bool initialized = false;
+    if (!initialized) {
+        static boost::locale::localization_backend_manager manager =
+            boost::locale::localization_backend_manager::global();
+        manager.select("icu");
+
+        static boost::locale::generator curLocale(manager);
+        utf8 = curLocale("en_US.UTF8");
+        initialized = true;
+    }
+
+    std::string normalized = boost::locale::normalize(name, boost::locale::norm_nfd, utf8);
+    // Non-locale aware lowercase
+    boost::algorithm::to_lower(normalized);
+
```

Comment:
looks like you were testing this and forgot to change?

Reply:
No, I left it open so it can be decided by all of us.  Locale-aware or not. Choose wisely ;-)

Reply:
According to  http://charette.no-ip.com:81/programming/doxygen/boost/namespaceboost_1_1algorithm.html#a1a318a47f9b4482c28552a62d0297a27 , it seems to be using std::locale() which is whatever the global locale is. 

And it seems like the global locale always defaults to  the "C" locale for C++ programs? (At least that's what it is set to for me, and other sources seem to say that the global locale should always be "C" )

Reply:
Right, C is usually the default, which is not UTF-8.  The goal is to be systematic (i.e. consistently using normalization), and then applying some effort on lower casing (where a perfect system for all encodings doesn't exist).  So long as we're consistent, it's best effort IMO.  I'm open to other takes on it.